api/routes.py

```diff
@@ -3,7 +3,7 @@
 from functools import update_wrapper, wraps
 
 import flask
-from flask import Response, make_response, redirect, request
+from flask import Response, g, make_response, redirect, request
```

Comment:
`g` doesn't appear to be used anymore.

`redirect`, which was already present, also appears not to be used.